app/assets/javascripts/surveys.js

```diff
@@ -49,7 +49,44 @@
       surveyType: 'url',
     },
     smallSurveys: [
-      
+      {
+        url: "https://signup.take-part-in-research.service.gov.uk/home?utm_campaign=" + window.location.pathname + "&utm_source=Hold_gov_to_account&utm_medium=gov.uk%20survey&t=GDS",
+        identifier: 'mar_ur_panel',
+        template: '<section id="user-satisfaction-survey" class="visible" aria-hidden="false">' +
+                  '  <div class="wrapper">' +
+                  '    <h1>Help improve GOV.UK</h1>' +
+                  '    <p class="right"><a href="#survey-no-thanks" id="survey-no-thanks">No thanks</a></p>' +
+                  '    <p><a href="javascript:void()" id="take-survey" target="_blank" rel="noopener noreferrer">Answer some questions about yourself to join the research community</a> This link opens in a new tab.</p>' +
+                  '  </div>' +
+                  '</section>',
+        frequency: 5,
+        activeWhen: function() {
+          function pathMatches() {
+            var pathMatchingExpr = new RegExp('/(?:'
+                + /government\/policies/.source
+                + /|government\/how-government-works/.source
+                + /|make-a-freedom-of-information-request/.source
+                + /|government\/collections\/open-government/.source
+                + /|government\/publications\/uk-open-government-national-action-plan-2016-18\/uk-open-government-national-action-plan-2016-18/.source
+                + /|government\/policies\/government-transparency-and-accountability/.source
+                + /|topic\/local-government\/transparency/.source
+                + /|government\/world/.source
```

Comment:
I think we'll need to split this survey into two as these last 6 paths want `"&utm_source=Improve_platform_basics` in the url instead of `"&utm_source=Hold_gov_to_account` like the first 7.